test/remote/remote_complete_test.rb

```diff
@@ -0,0 +1,27 @@
+require 'test_helper'
+
+class RemoteCompleteTransactionTest < Test::Unit::TestCase
+  def setup
+    @environment = Spreedly::Environment.new(remote_test_environment_key, remote_test_access_secret)
+  end
+
+  def test_successful_refund_full_amount
```

Comment:
The test doesn't appear to be testing refunds. Should this be named differently?

Reply:
ugh yes. It was a copy of structure and highly modified 👍 The copy-pasta monster got me.